projects/packages/my-jetpack/_inc/components/products-table-view/style.scss

```diff
@@ -0,0 +1,91 @@
+@import '@wordpress/dataviews/build-style/style.css';
+@import '@wordpress/components/build-style/style.css';
+
+:root {
+    --font-size: 13px;
+    --tag-color: #DAFFDC;
+}
+
+button.dataviews-view-list__item {
+    display: none;
+}
+
+svg.table-view-icon {
+    width: 52px;
+    height: 52px;
+}
+
+button[aria-controls="dataviews-view-config-dropdown-0"] {
+    display: none;
+}
+
+div.dataviews__view-actions {
+    padding-left: 24px;
+    padding-right: 24px;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+div.dataviews-filters__container {
+    padding-left: 24px;
+}
+
+button.components-button.is-secondary {
+    padding: calc( var(--spacing-base) / 2 ) var(--spacing-base) !important;
```

Comment:
Would it be possible to get rid of important with more specificity? I know it's not always possible with how we've been using CSS Modules, but it helps to have a more predictable code and behavior.

Reply:
Not sure why this was required before 🤔 

Seems to be working without it now, thanks!